### ISSUE TITLE
IOS App file is missing!

### ISSUE THREAD

**Issue by iMonZ** [bennyboer/bbb_app#167]
The IOS files (.ipa) are missing from the release section.

**Comment by bennyboer** [comment]
Can you elaborate on the benefit this would bring?

**Comment by iMonZ** [comment]
> Can you elaborate on the benefit this would bring?

The apk file is available too and on iOS you can sideload apps too!

**Comment by bennyboer** [comment]
Yeah but isn't that comparatively hard to do? I mean if you're able to sideload apps on iOS you might just want to build it yourself. We included the APK for people that don't want to use the play store, which is an officially supported way on Android devices. I don't know people that buy Apple products and then jailbreak it just to not use any of the Apple services (due to privacy concerns for example).

**Comment by iMonZ** [comment]
You don’t need to jailbreak the iPhone!
It’s just because you don’t use an iPhone device!
Then apps can be officially sideloaded with tools like AltServer or AppDB! You don’t even need a Mac for that!

**Comment by bennyboer** [comment]
I suppose, but I just don't see the benefit when it is already available on the App Store.

**Comment by iMonZ** [comment]
> I suppose, but I just don't see the benefit when it is already available on the App Store.

As example for devices with MDM.
You can't just install apps from the App Store but you can side-load them very easy

**Comment by bennyboer** [comment]
I see! We will check later on to setup a CI pipeline to do this!

**Comment by iMonZ** [comment]
> I see! We will check later on to setup a CI pipeline to do this!

Perfect this would make everything more equal
